cfpb/cfgov-refresh
# Find-a-housing-counselor: Ignore map on voiceover

## Changes

- Adds `aria-hidden="true"` attribute to housing counselor mapbox map.
- Adds `z-index` setting to mapbox map container CSS to contain layering of zoom buttons.
- Converts tabs to spaces in CSS file.

## Testing

1. Pull branch.
1. Visit /find-a-housing-counselor/

Mega menu bug:
1. Expand menu and see that zoom buttons in map don't overlap the mega menu.
2. Compare to http://consumerfinance.gov/find-a-housing-counselor and see that zoom buttons in map DO overlap the mega menu.

Accessibility:
1. Turn on VoiceOver with cmd + f5.
1. Click on page and hit tab to get focus on "HOME".
1. Hit ctrl+option+right arrow keys repeatedly to see that voiceover does not enter the map.
1. Compare to http://consumerfinance.gov/find-a-housing-counselor and see that VoiceOver does enter map tiles.
